### PR TITLE
ADD STEPS CHECKER

### PR DESCRIPTION
**Pull request by danilowhk**
Added measure output for each test

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: tests/test_zk_evm_erc20.py [hunk 1]
```diff
@@ -101,48 +104,69 @@ async def test_erc20(
...
+        print(res.call_info.execution_resources)
+
         # Call transfer function
         print("CALLING transfer")
         res = await zk_evm.execute_at_address(
             address=evm_contract_address,
             calldata=[int(b, 16) for b in wrap(params["transfer"], 2)],
         ).execute(caller_address=1)
 
+        print("ERC20 trasnfer tx infos")
```
Comment (Eikix) [inline tests/test_zk_evm_erc20.py:157]: typo

File: tests/test_zk_evm_erc20.py [hunk 2]
```diff
@@ -101,48 +104,69 @@ async def test_erc20(
             evm_address=evm_contract_address, starknet_address=starknet_contract_address
         ).execute(caller_address=1)
 
+        print("ERC20 Contract initiation tx infos")
```
Comment (Eikix) [inline tests/test_zk_evm_erc20.py:107]: initialization?

File: tests/test_zk_evm.py [hunk 3]
```diff
@@ -1593,12 +1602,18 @@ async def test_execute_at_address(self, zk_evm, eth, params):
             evm_address=evm_contract_address, starknet_address=starknet_contract_address
         ).execute(caller_address=1)
 
+        print("Contract Initiation tx infos")
```
Comment (Eikix) [inline tests/test_zk_evm.py:1605]: typo I think

**Review by Eikix — CHANGES_REQUESTED** [review]
Just a few typos and LGTM

File: tests/test_zk_evm_erc20.py (hunk 2 above)
Comment (danilowhk) [reply tests/test_zk_evm_erc20.py:107]: So right now, the "constructor" is working in 2 stages. Deployment and running the initial constructor code. The ideal is to have the initial constructor code run during deployment time

File: tests/test_zk_evm.py (hunk 3 above)
Comment (danilowhk) [reply tests/test_zk_evm.py]: fix pushed

File: tests/test_zk_evm.py [hunk 4]
```diff
@@ -1602,7 +1602,7 @@ async def test_execute_at_address(self, zk_evm, eth, params):
             evm_address=evm_contract_address, starknet_address=starknet_contract_address
         ).execute(caller_address=1)
 
-        print("Contract Initiation tx infos")
+        print("Contract initiation tx infos")
```
Comment (Eikix) [inline tests/test_zk_evm.py]: initialization I think? initiation: Initiation is a rite of passage marking entrance or acceptance into a group or society. (wikipedia)

**Review by Eikix — CHANGES_REQUESTED** [review]
last typo and lgtm

File: tests/test_zk_evm.py (hunk 4 above)
Comment (danilowhk) [reply tests/test_zk_evm.py]: Oh yeah haha my bad

**Review by Eikix — CHANGES_REQUESTED** [review]
Sorry, last PR introduced a couple of conflicts;)!
Do you mind rebasing on main, then I'll be able to approve your PR.

**Review by Eikix — APPROVED** [review]
lgtm, awaiting tests

**Pull request merged**